### DISCUSSION TITLE
JSON Schema

### DISCUSSION THREAD

**Discussion by catamaican** [vueform/vueform#80] [Ideas]
### Describe the feature

Any chance of integrating schema based on https://json-schema.org ?

Thanks

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a plugin?

### Final checks

- [X] Read the [contribution guide](https://vueform.com/community/contribution-guide).
- [X] Check existing [discussions](https://github.com/vueform/vueform/discussions) and [ideas](https://github.com/vueform/vueform/issues/categories/ideas).

**Comment by adamberecz** [comment]
I would be open to discover the possibility if anyone has enough experience with JSON Schema and it's easily doable.
